src/statisticBlock.js

```diff
@@ -111,13 +109,35 @@ class StatisticBlock extends Component {
       let clientWidth = this.state.clientWidth;
       let clientHeight = this.state.clientHeight;
       let childHeight = 0;
+      let childWidth = 0;
+      let containerElm = this.refs.parent;
+      let childrenElm =containerElm.children;
+      let childrenCombinedWidth = 0;
+      let childrenHeight = childrenElm[0].clientHeight;
+      for (let e of childrenElm) {
```

Comment:
If it's the dimensions you're trying to calculate then take a look at [getBoundingClientRect](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect).